default/buildings.txt

```diff
@@ -611,44 +595,42 @@ BuildingType
     ]
     EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
-        EffectsGroup
+	    EffectsGroup
```

Comment:
Tab added unnecessarily, we avoid tabs in the scripts preferring 4 spaces consistently